closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -109,7 +378,7 @@ JNIEXPORT jlong JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_DigestCreateCon
 
     if (copyContext != 0) {
         EVP_MD_CTX *contextToCopy = ((OpenSSLMDContext*) copyContext)->ctx;
-        EVP_MD_CTX_copy_ex(ctx, contextToCopy);
+        (*OSSL_MD_CTX_copy_ex)(ctx,contextToCopy);
```

Comment:
nit: space missing after comma

Reply:
Yes, this is very nit-picky but I will fix it for you.

Reply:
Done